src/screens/UserPortal/Events/Events.spec.tsx

```
@@ -1404,6 +1429,8 @@ describe('Testing Events Screen [User Portal]', () => {
     await waitFor(() => {
```

ESLint

[error] Insert `⏎······`

(prettier/prettier)